libdnf/utils/fs.cpp

```diff
@@ -0,0 +1,29 @@
+#include <sys/stat.h>
+
+#include <filesystem>
+
+#include "fs.hpp"
+
+
+namespace libdnf::utils::fs {
+
+
+void makedirs_for_file(std::string file_path) {
```

Comment:
Don't need to copy `file_path` string. It is used only by `weakly_canonical()` which takes const reference.
`void makedirs_for_file(const std::string & file_path);`

Reply:
I need to keep an eye on these issues to avoid creating them.
Fixed.